middlewares/validators/tasks.js

```diff
@@ -77,6 +84,13 @@ const updateTask = async (req, res, next) => {
           [NEELAM]: joi.number().optional(),
         })
         .optional(),
+      taskLevel: joi
+        .object()
+        .keys({
+          category: joi.string().optional(),
+          level: joi.number().optional(),
+        })
+        .optional(),
```

Comment:
This should also be fixed as we are not using `taskLevel` object anymore, category and level should be individually validated.

Reply:
I don't know why it's there, I pulled Muralidhar's PR some days back so I think when I created a new branch, those changes still were there, let me remove it